lib/services/services.c

```diff
@@ -932,16 +932,16 @@ execute_metadata_action(svc_action_t *op)
 
     if (op->agent == NULL) {
         crm_err("meta-data requested without specifying agent");
```

Comment:
This list is an example where we could think of having those errors in as well.
As that not to end in escalated heap-allocation, string-comparison, ... we could as well think over some flags
to indicate if it is owned by the op-object and thus should be freed, some kind of type/level so that we can
use that mechanism as well to pass along data that is rather useful for debugging or if it should be shown
in the enduser-tooling, ...